index.html

```diff
@@ -18,6 +18,14 @@
     </div>
     <nav>
       <img id="burgermenu" src="./img/burger.png" alt="">
+      <div id="menu">
+        <ul id="menu-container">
+          <li class="menu-item"><a class="menu-style" href="#">Portfolio</a></li>
+          <li class="menu-item"><a class="menu-style" href="#">About</a></li>
+          <li class="menu-item"><a class="menu-style" href="#">Contact</a></li>
+          <li class="menu-item"><img id="mail-icon" src="/img/icons/icons8-mail-48.png" alt=""></li>
+        </ul>
+      </div>
```

Comment:
you have a gap in 800px widescreen, I suggest fixing that so your portfolio will display on any device smoothly
![image](https://user-images.githubusercontent.com/61790587/165862634-958b790f-5ecb-45b6-9c46-6e7d2b477a20.png)